components/clarity-vscode/package.json

```diff
@@ -77,6 +77,26 @@
             ],
             "default": "verbose",
             "description": "Traces the communication between VS Code and the web-extension language server."
+          },
+          "clarity-lsp.completion": {
+            "type": "boolean",
+            "default": true,
+            "description": "Allow auto completion for native and custom functions"
+          },
+          "clarity-lsp.hover": {
+            "type": "boolean",
+            "default": true,
+            "description": "Show documentation for native keywords and method on Hover"
+          },
+          "clarity-lsp.documentSymbols": {
+            "type": "boolean",
+            "default": false,
```

Comment:
False by default. This feature was too unstable lately compared to the benefits it brings. (Even though issues should be fixed now, it seems safer for now)

Reply:
is it possible / would it make sense to enable this by default only on pre-releases.
it'd be great to have some eyes on experimental features if developers are willing to use pre-release software.